README.rst

```diff
@@ -537,6 +537,19 @@ Choose from the list of available rules:
 
   *Risky rule: risky if the ``ereg`` funcion is overridden.*
 
+* **escape_implicit_backslashes**
+
+  Escape implicit backslashes in strings and heredocs to increase code
+  readability.
+
+  Configuration options:
+
+  - ``double_quoted`` (``bool``): whether to fix double-quoted strings; defaults to
+    ``true``
+  - ``heredoc_syntax`` (``bool``): whether to fix heredoc syntax; defaults to ``true``
+  - ``single_quoted`` (``bool``): whether to fix single-quoted strings; defaults to
```

Comment:
why one would like to use it, pointing to already pointed example of this config:
```diff
-echo '<?php namespace A\B;  class C {}'
+echo '<?php namespace A\\B;  class C {}'
```

`\` has no special meaning inside single_quoted text except of escaping the quote. so `'\n'` doesn't make new line even if `\` is not double backslashed

Reply:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3155#issuecomment-336855147 ?

Reply:
you see, it's not intuitive and extra edgy case ;P

Reply:
Please be more explicit on what you would like me to do.

I think https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3155#issuecomment-336855147 is a valid point, but I agree it's edgy and is in fact disabled by default.

If you want me to delete it, I'll delete it.
If you want me to add more description (on this flag, I already know I have to improve the general summary), I'll add more description.

As of yet, I have no idea what I have to do 🦆

Reply:
at least descr would be nice ;)